### ISSUE TITLE
toolchain: ensure SIRIUS uses LIBVDWXCROOT

### ISSUE THREAD

**Issue by drhpc** [cp2k/cp2k#2841]
I am not 100% sure when it is needed, but I had to insist for SIRIUS build to use the proper libvdwx location. I don't see how it would hurt to always do this.

```
--- tools/toolchain/scripts/stage8/install_sirius.sh.orig	2023-06-26 09:12:47.440695921 +0200
+++ tools/toolchain/scripts/stage8/install_sirius.sh	2023-06-26 09:18:20.136293662 +0200
@@ -15,6 +15,12 @@
 source "${INSTALLDIR}"/toolchain.conf
 source "${INSTALLDIR}"/toolchain.env
 
+# Did this change recently? Check!
+# CMake code for finding the lib is broken and uses inconsistent
+# variables for include and lib search, while ignoring what pkg-config
+# has to offer, too. Stupid as a brick. But a brick is at least stable.
+export VDWXCROOT=$LIBVDWXCROOT
+
 if [ "$MPI_MODE" = "no" ]; then
   report_warning $LINENO "MPI is disabled, skipping sirius installation"
   echo 'with_sirius="__FALSE__"' >> ${BUILDDIR}/setup_sirius
EOT
```

**Comment by drhpc** [comment]
It seems my build currently works well without this patch. I needed it for CP2K 7.1. So maybe the issue can just be closed …

**Comment by mkrack** [comment]
Thanks for reporting. Many toolchain fixes have been applied since version 7.1 and it is quite likely that this issue was resolved.
